test/services/MemberDetailsUploadValidatorSpec.scala

```diff
@@ -253,7 +253,7 @@ class MemberDetailsUploadValidatorSpec extends BaseSpec with TestValues {
             ValidationError("F3", DateOfBirth, "memberDetails.dateOfBirth.upload.error.future"),
             ValidationError("C4", LastName, "memberDetails.lastName.upload.error.invalid"),
             ValidationError("F5", DateOfBirth, "memberDetails.dateOfBirth.upload.error.future"),
-            ValidationError("E5", NoNinoReason, "noNINO.upload.upload.error.invalid"),
+            ValidationError("E5", NoNinoReason, "error.textarea.invalid"),
```

Comment:
In this case, the validation is agains a value in the CSV file and not against a value of the text field on the UI. It would be worthwhile to ask design / content design if they are still happy with the error message "The submitted text .... " here.